packages/block-editor/src/components/index.js

```diff
@@ -108,6 +109,7 @@ export {
 	useClipboardHandler as __unstableUseClipboardHandler,
 } from './copy-handler';
 export { default as DefaultBlockAppender } from './default-block-appender';
+export { default as __experimentalDuotoneFilter } from './duotone-filter';
```

Comment:
What is this export do?  How folks are supposed to use this?

Reply:
That's just the generated SVG and stylesheet, so now that we're not using it in the image block directly, it can be moved just like duotone-filter.php